docs/_source/components-overview/post_processing/pipeline/pipeline_steps.rst

```diff
@@ -206,7 +206,10 @@ type of the rerun specifying the logic and changes, and the cluster name.
     conv_bdy_weight        caution         it disabled the convective_bdy_weight where this caused segmentation faults (this avoids a bug in the old MESA version r11701)
     dedt_energy_eqn        caution         it enables MESA's dedt-form of the energy equation for numerical stability during rapid (superthermal) mass transfer
     dedt_hepulse           caution         it enables MESA's dedt-form of the energy equation for rapid mass transfer; at stripped HeZAMS, several MLT++ changes, v_flag and lnPgas_flag set to .true., and convective_bdy_weight disabled to help with stripped He star superadiabatic envelopes, pulsations, and WD cooling
-    LBV_wind               default in v3+  it turns on LBV winds when crossing the Humphreys-Davidson limit as intended (due to a bug this was only applied after a retry); additionally, there are reruns `LBV_wind+thermohaline_mixing`, `LBV_wind+dedt_energy_eqn`, which combine the two rerun types
+    LBV_wind               default in v3+  it turns on LBV winds when crossing the Humphreys-Davidson limit as intended (due to a bug this was only applied after a retry); additionally, there are reruns `LBV_wind+thermohaline_mixing`, `LBV_wind+dedt_energy_eqn`, which combine the two rerun types. Any additional changes to these reruns are described here as LBV_wind+rerun_type
     no_age_limit           default in v3+  it allows low mass stars to evolve beyond the age of the universe, which is needed for grids where we jump on past ZAMS; additionally, there are reruns `no_age_limit+thermohaline_mixing` and `no_age_limit+dedt_energy_eqn`, which combine the two rerun types
+    LBV_wind+dedt          caution         it enables MESA's dedt-form of the energy equation for numerical stability during rapid (superthermal) mass transfer and sets lnPgas_flag to .true. for numerical stability. Also disabled convective_bdy_weight as a degenerate core is forming (as probed by the central Coulomb coupling parameter) to avoid segmentation faults.
```

Comment:
Is that anything different from `LBV_wind+dedt_energy_eqn`?
I think, at the moment this table becomes less and less understandable for any person how didn't follow the steps we made. Hence, maybe we should clean this all up to may remove all the stuff we discarded.